Games/types/Mafia/roles/cards/Bloodthirsty.js

```diff
@@ -0,0 +1,47 @@
+const Card = require("../../Card");
+
+module.exports = class Bloodthirsty extends Card {
+
+    constructor(role) {
+        super(role);
+
+        role.data.blood = 50;
+
+        this.listeners = {
+            "rolesAssigned": function () {
+                this.player.queueAlert(`You have ${this.data.blood}% blood left!`);
+            },
+            "actionsNext": function () {
+                if (!this.player.alive)
+                    return;
+
+                if (this.game.getStateName() != "Night")
+                    return;
+
+                this.data.blood -= 25;
+                if (this.data.blood <= 0){
+                    this.player.kill("bloodthirst", this.player);
+                }
+                if (this.data.blood > 0){
+                    if (this.data.blood <= 25){
+                        this.player.queueAlert(`You hunger for blood. You now have ${this.data.blood}% blood left! If you don't kill anybody before the next day, you will die!`);
+                    } else {
+                        this.player.queueAlert(`You hunger for blood. You now have ${this.data.blood}% blood left!`);
+                    }
+                }
+            },
+            "death": function (player, killer, deathType) {

```

Comment:
check death type is not lynch

Reply:
Does the village vote attribute the death cause to a specific person? I thought it was just a general death type that didn't have a specific killer attached to it.

Reply:
village vote gives to the first person who voted